File: src/ashwini/model/initial_params.jl
include("conductivity.jl")

# Initial value of dₖ₁

function dₙ₁₀(I)
    F*lₙ*I*(1/k₂ₙ(c₂₀) - 1/σ₁ₙ)/(4*R*T)
end

function dₚ₁₀(I)
    F*lₚ*I*(1/k₂ₚ(c₂₀) - 1/σ₁ₚ)/(4*R*T)
end

File: scripts/new_ashwini/model/initial_params.m
% Include conductivity.m
run('conductivity.m')

% Initial value of d_k_1

% d_n_1_0 function
function result = d_n_1_0(I)
    result = F*l_n*I*(1/k_2_n(c_2_0) - 1/sigma_1_n)/(4*R*T);
end

% d_p_1_0 function
function result = d_p_1_0(I)
    result = F*l_p*I*(1/k_2_p(c_2_0) - 1/sigma_1_p)/(4*R*T);
end
